src/main/resources/application.yml

```
@@ -0,0 +1,41 @@
+discodeit:
+  storage:
+    type: local
+    local:
+      root-path: .discodeit/storage
+
+spring:
+  jpa:
+    open-in-view: false
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        default_batch_fetch_size: 100
+  profiles:
+    active: local
+  servlet:
+    multipart:
+      enabled: true
+      max-file-size: 5MB  # 업로드할 파일 최대 크기
+      max-request-size: 10MB
+
+
+  datasource:
+    url: jdbc:postgresql://localhost:5432/discodeit
+    username: discodeit_user
+    password: discodeit1234
+    driver-class-name: org.postgresql.Driver
+    hikari:
+      maximum-pool-size: 10
+
+
+logging:
+  level:
+    root:
+    org.springframework.web: DEBUG
+    org.springframework.web.servlet: DEBUG
+    org.hibernate.sql: debug
+    org.hibernate.orm.jdbc.bind: trace
+
```

```suggestion
logging:
  level:
    root:
      org.springframework.web: DEBUG
      org.springframework.web.servlet: DEBUG
      org.hibernate.sql: debug
      org.hibernate.orm.jdbc.bind: trace

```

들여쓰기가 빠졌네요. 한번 더 들여쓰기가 되어야 합니다!
추가로, org.springframework.web는 왜 DEBUG 레벨로 설정하셨는지 궁금하네요!

---

어느 부분에서 오류가 나는지, 어떤식으로 동작하는지 제대로 알고 싶어서 DEBUG 설정해놓았었습니다. 
현재는 WARN으로 변경했는데 괜찮을까요?